home/models.py

```diff
@@ -264,6 +266,16 @@ def average_rating(self):
     def get_absolute_url(self):
         return reverse("professor", kwargs={"slug": self.slug})
 
+    @staticmethod
+    def find_similar(professor_name):
+        SIMILARITY_TOLERANCE = 70
+        similar_professors = []
+        for professor in Professor.verified.all():
+            if fuzz.ratio(professor_name, professor.name) > SIMILARITY_TOLERANCE:
+                similar_professors.append(professor)
+
+        return similar_professors
+
```

Comment:
if this is going to be a method on `Professor`, then `tolerance` should be a parameter, probably without a default value. The cutoff of 70 is too specific and arbitrary to justify a static method solely for that cutoff.